docs/runtime_options.md

```diff
@@ -158,13 +158,25 @@ automatically.
 * `grouping` - in scenario when accounted for events are used to calculate derivative metrics, it is reasonable to
 measure them in transactional manner: all the events in a group must be accounted for in the same period of time. Keep
 in mind that it is impossible to group more events that there are counters available.
+* `PMU` - Performance Monitoring Unit
+* `uncore events` - events which can be counted by PMUs outside core.
+
+#### Getting config values
+Using perf tools:
+* Identify the event in `perf list` output. 
+* Execute command: `perf stat -I 5000 -vvv -e [EVENT_NAME]`
+* Find `perf_event_attr` section on `perf stat` output
+
```

Comment:
Could you provide here an example of perf stat output and config prepared from  this output?